ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh

```diff
@@ -126,6 +127,8 @@ oc adm wait-for-stable-cluster
 
 # create new admin.kubeconfig
 oc config new-admin-kubeconfig > "${SHARED_DIR}/admin.kubeconfig"
+oc --kubeconfig="${SHARED_DIR}/admin.kubeconfig" whoami || true
+sleep 2
```

Comment:
By the sleeping, does it mean `new-admin-kubeconfig` would generally take some time, so better to sleep a bit before using the new kubeconfig? Thanks!